enjoy/src/main/youtubedr.ts

```diff
@@ -255,6 +257,22 @@ class Youtubedr {
   abortDownload() {
     this.abortController?.abort();
   }
+
+  /**
+   * 代理环境变量配置
```

Comment:
Keep English only in the codebase please.

Reply:
Okay, thanks for reminding me, I'll keep it in mind.